quick_tests.py

```diff
@@ -261,10 +262,13 @@ def print_filter(d_filter):
 
     print "Clear Router Diagnostics"
     print "========================"
-    transceiver.clear_router_diagnostic_counters(0, 0)
-    router_diagnostics = transceiver.get_router_diagnostics(0, 0)
-    print router_diagnostics.registers
-    print ""
+    transceiver.clear_router_diagnostic_counters(
+        0, 0, counter_ids=[router_diagnostics._REGISTERS.LOC_PP.value,
```

Comment:
The short answer is Yes.  The long answer is Yes, Yes it should.

From: Alan Stokes [mailto:notifications@github.com]
Sent: 10 August 2015 10:29
To: SpiNNakerManchester/SpiNNMan <SpiNNMan@noreply.github.com>
Cc: Andrew Rowley <Andrew.Rowley@manchester.ac.uk>
Subject: Re: [SpiNNMan] Speed up data transfer 2 (#34)


In quick_tests.py<https://github.com/SpiNNakerManchester/SpiNNMan/pull/34#discussion_r36614336>:

> @@ -261,10 +262,13 @@ def print_filter(d_filter):

>

>      print "Clear Router Diagnostics"

>      print "========================"

> -    transceiver.clear_router_diagnostic_counters(0, 0)

> -    router_diagnostics = transceiver.get_router_diagnostics(0, 0)

> -    print router_diagnostics.registers

> -    print ""

> +    transceiver.clear_router_diagnostic_counters(

> +        0, 0, counter_ids=[router_diagnostics._REGISTERS.LOC_PP.value,

shouldnt the router_diagnostics be a enum inside spinnman, to a point where its public assessable (as a front end user shouldnt need to access a private enum to get/clear a specfic router counter.

—
Reply to this email directly or view it on GitHub<https://github.com/SpiNNakerManchester/SpiNNMan/pull/34/files#r36614336>.